### PR TITLE
III-3501 Create the offer popularity table

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added migration to create the offer popularity table
---
Ticket: https://jira.uitdatabank.be/browse/III-3501

### CODE REVIEW

**Review by jdrieghe — APPROVED** [review]
Looks clean

File: app/Migrations/Version20201022154706.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            [
+                'notnull' => true,
+            ]
+        );
+
+        $table->addIndex(
+            [
+                'offer_id',
+            ],
+            'offer_id_index'
```
Comment (bertramakers) [inline app/Migrations/Version20201022154706.php:49]: Maybe we should set this as the primary key? Then it will always get an index as well.

**Review by bertramakers — APPROVED** [review]

File: app/Migrations/Version20201022154706.php (hunk 1 above)
Comment (LucWollants) [reply app/Migrations/Version20201022154706.php:49]: Then we also have a unique constraint on that column, that is not an issue for the upload?

Comment (bertramakers) [reply app/Migrations/Version20201022154706.php:49]: As noted in the meeting notes (but maybe a bit buried) the table will be truncated every time before new data is inserted.

**Pull request merged**